rules/windows/process_creation/proc_creation_win_sysinternals_psexec_remote_execution.yml

```diff
@@ -0,0 +1,28 @@
+title: Potential PsExec Remote Execution
+id: ea011323-7045-460b-b2d7-0f7442ea6b38
+status: experimental
+description: Detects potential psexec command that initiate execution on a remote systems via common commandline flags used by the utility
+references:
+    - https://docs.microsoft.com/en-us/sysinternals/downloads/psexec
+    - https://www.poweradmin.com/paexec/
+    - https://www.fireeye.com/blog/threat-research/2020/10/kegtap-and-singlemalt-with-a-ransomware-chaser.html
+author: Florian Roth (Nextron Systems), Nasreddine Bencherchali (Nextron Systems)
+date: 2023/02/28
+tags:
+    - attack.resource_development
+    - attack.t1587.001
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        # Accepting EULA in commandline - often used in automated attacks
+        CommandLine|contains|all:
+            - 'accepteula'
```

Comment:
With accepteula in the `all`  contains , it will match only the automated attacks.

Reply:
To keep it psexec related we need the accepteula. Once we get more data, I can create another one that's more generic.

Reply:
OK, when a generic will be add the title will be updated.